rllib/policy/eager_tf_policy.py

```diff
@@ -729,12 +733,35 @@ def set_state(self, state: PolicyState) -> None:
             super().set_state(state)
 
         @override(Policy)
-        def export_checkpoint(self, export_dir):
-            raise NotImplementedError  # TODO: implement this
-
-        @override(Policy)
-        def export_model(self, export_dir):
-            raise NotImplementedError  # TODO: implement this
+        def export_model(self, export_dir, onnx: Optional[int] = None) -> None:
```

Comment:
`export_checkpoint` entirely defined in parent class (Policy).

`export_model` previously not implemented!